test/src/data/new-subjects.js

```diff
@@ -50,7 +50,7 @@ created_at: "2022-05-27T12:13:44.311511-05:00"
 },
 ad_html: "<strong>Assign homework and readings synced with OpenStax textbooks</strong>",
 link_text: "Learn more",
-link_href: "https://dev.openstax.org/openstax-tutor"
+link_href: "https://dev.openstax.org/openstax-thingy"
```

Comment:
is this a thing?

Reply:
I wanted to leave the data item in but not have any instance of "tutor" in the source.